deepdiff/serialization.py

```diff
@@ -254,10 +256,40 @@ def find_class(self, module, name):
         # Forbid everything else.
         raise ForbiddenModule(FORBIDDEN_MODULE_MSG.format(module_dot_class)) from None
 
+    def persistent_load(self, persistent_id):
+        if persistent_id == "<<NoneType>>":
```

Comment:
I'm not super happy with this solution but I didn't find a better solution to the problem.